packages/codemodel.go/src/gocodemodel.ts

```diff
@@ -769,9 +769,9 @@ export interface LiteralValue {
   literal: any;
 }
 
-// StandardType is a type from the Go standard library (excluding time.Time)
-export interface StandardType {
-  // this is the fully-qualified name (e.g. io.Reader)
+// PackageType is a type from some package, e.g. the Go standard library (excluding time.Time)
+export interface PackageType {
```

Comment:
Would QualifiedType be more to the point i.e., the crucial detail is that we must qualify references to this type?

Reply:
I like it and it aligns with the terminology used in the [language spec](https://go.dev/ref/spec#Qualified_identifiers).